POEApi.Model/Data.xml

```diff
@@ -1694,10 +1704,11 @@
       <Item name="The Warlord"/>
       <Item name="The Watcher"/>
       <Item name="The Web"/>
+      <Item name="The Wilted Rose"/>
       <Item name="The Wind"/>
       <Item name="The Witch"/>
-      <Item name="The Wolf"/>
       <Item name="The Wolf's Shadow"/>
+      <Item name="The Wolf"/>
```

Comment:
Nit: You're sorting this after `The Wolf's Shadow`, but `The Master Artisan` is being moved to before `The Master`, which doesn't seem consistent.  I usually order by "nothing before something", so I would expect `The Wolf` to sort before `The Wolf's Shadow`.